modules/logging-loki-gui-install.adoc

```diff
@@ -40,15 +40,81 @@ This option sets the `openshift.io/cluster-monitoring: "true"` label in the Name
 +
 If the approval strategy in the subscription is set to *Automatic*, the update process initiates as soon as a new Operator version is available in the selected channel. If the approval strategy is set to *Manual*, you must manually approve pending updates.
 
-The Operators should now be available to all users and projects that use this cluster.
+. Create a secret.
```

Comment:
Would it make sense to split the "Installing a Loki log store" into three parts instead of the current two:

- install operator using Web-UI
- install operator using CLI
- configure LokiStack after installation

Currently the "configure LokiStack" part is present in both Web-UI and CLI installations but the one in Web-UI goes into more detail (also talking about the Secret and, possibly, ClusterLogging configuration).

Reply:
Created a Jira to follow up with this https://issues.redhat.com/browse/OBSDOCS-674